### PR TITLE
Step to install keys through cocoapods-keys plugin

### PR DESCRIPTION
**Pull request by igorcferreira**
TagCheck

The goal of this step is to allow the CI to inject keys into an iOS project using [cocoapods-keys plugin](https://www.github.com/orta/cocoapods-keys). So, the value is securely stored into source code.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/cocoapod-key/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      to match the order and size of the `values` property
+    is_expand: true
+    is_reqired: true
+    summary: The list of keys that will be set. With `|` as a separator
+    title: Keys
+- opts:
+    description: The list of values that will be set. With `|` as a separator. It
+      needs to match the order and size of the `keys` property
+    is_expand: true
+    is_reqired: true
```
Comment (BirmacherAkos) [inline steps/cocoapod-key/1.0.0/step.yml]: Please change it to is_required

File: steps/cocoapod-key/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+is_always_run: false
+is_skippable: false
+run_if: .IsCI
+inputs:
+- opts:
+    description: In case that your workspace has multiple projects, and multiple targets,
+      a `pod key` command may fail because of the conflict in the configuration list.
+      This solves this issue
+    is_expand: true
+    is_reqired: false
```
Comment (BirmacherAkos) [inline steps/cocoapod-key/1.0.0/step.yml]: Please change it to is_required

File: steps/cocoapod-key/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    summary: The name of the project. It will be used to resolve project conflict,
+      in the case of multiple targets in the workspace
+    title: Project name
+  project_name: ""
+- keys: ""
+  opts:
+    description: The list of keys that will be set. With `|` as a separator. It needs
+      to match the order and size of the `values` property
+    is_expand: true
+    is_reqired: true
```
Comment (BirmacherAkos) [inline steps/cocoapod-key/1.0.0/step.yml]: Please change it to is_required

File: steps/cocoapod-key/1.0.0/step.yml (hunk 1 above)
Comment (igorcferreira) [reply steps/cocoapod-key/1.0.0/step.yml]: Sure will. Sorry @BirmacherAkos for the typo. I'll check if I can automate this checks.

Comment (igorcferreira) [reply steps/cocoapod-key/1.0.0/step.yml]: @BirmacherAkos I fixed the issues, and created a new PR (#1908).

**Comment by BirmacherAkos** [comment]
Hi @igorcferreira 

Sadly we had to remove your step version because of a typo in the step.yml.
You wrote is_reqired instead of  is_required
Here
https://github.com/bitrise-io/bitrise-steplib/pull/1904/files#diff-1e6dca9db9ad0bb4ae08e018cb23e441R32

here
https://github.com/bitrise-io/bitrise-steplib/pull/1904/files#diff-1e6dca9db9ad0bb4ae08e018cb23e441R42

and here:
https://github.com/bitrise-io/bitrise-steplib/pull/1904/files#diff-1e6dca9db9ad0bb4ae08e018cb23e441R49

We got some user feedbacks that it causes an error with older CLI versions.

---

Could you please create a new patch with the fix and share the step again?
Sorry for the inconvenience.

**Comment by igorcferreira** [comment]
@BirmacherAkos I fixed the step and created a [new PR](https://github.com/bitrise-io/bitrise-steplib/pull/1908).

**Pull request merged**